python/TestHarness/testers/PetscJacobianTester.py

```diff
@@ -18,7 +18,7 @@ class PetscJacobianTester(RunApp):
     def validParams():
         params = RunApp.validParams()
         params.addParam('ratio_tol', 1e-8, "Relative tolerance to compare the ration against.")
-        params.addParam('difference_tol', 1e-8, "Relative tolerance to compare the difference against.")
+        params.addParam('difference_tol', 1e-7, "Relative tolerance to compare the difference against.")
```

Comment:
Users of `PetscJacobianTester` basically always have to manually set a value for `difference_tol` currently because 1e-8 is too tight. 1e-7 is probably too tight as well, but at least it's a little better. In practice, it is the `ratio_tol` that really matters for communicating whether the hand-coded (or AD) Jacobian is accurate. We should probably consider removing this parameter altogether